frontend/public/storage/components/storage-overview/storage-overview.jsx

```diff
@@ -38,6 +39,10 @@ const CONSUMERS_SLCLASSES_REQUESTED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over
 const CONSUMERS_SLCLASSES_USED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over_time(kubelet_volume_stats_used_bytes[1h]) * on (namespace,persistentvolumeclaim) group_left(storageclass) kube_persistentvolumeclaim_info) by (storageclass))))[10m:1m]';
 const CONSUMERS_PODS_REQUESTED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over_time(kube_persistentvolumeclaim_resource_requests_storage_bytes[1h]) * on (namespace,persistentvolumeclaim) group_left(pod) kube_pod_spec_volumes_persistentvolumeclaims_info) by (pod))))[10m:1m]';
 const CONSUMERS_PODS_USED_CAPACITY_QUERY = '(sort(topk(5, sum(avg_over_time(kubelet_volume_stats_used_bytes[1h]) * on (namespace,persistentvolumeclaim) group_left(pod) kube_pod_spec_volumes_persistentvolumeclaims_info) by (pod))))[10m:1m]';
+
+const CONSUMERS_VMS_REQUESTED_CAPACITY_QUERY = '';
```

Comment:
what are the empty queries for?

Reply:
for the VMs dropdown in top consumers card

Reply:
why are they empty?

Reply:
the query is not ready till yet..

Reply:
ok, can you mark this PR WIP then?

Reply:
is it ok if the query can be sent with other updated queries in a followup PR ?

Reply:
It is not much useful without it. IMO it is better to wait for the query.

Reply:
alright :+1: